cinema/tests/test_movie_api.py

```diff
@@ -157,3 +155,12 @@
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedMovieApiTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_auth_required(self):
+        res = self.client.get(MOVIE_URL)
```

Comment:
Good job on testing the behavior of the API when the user is not authenticated. It's important to ensure that unauthenticated users can't access data they're not supposed to.